test/checkpoint/test_checkpoint.h

```diff
@@ -45,7 +45,7 @@
  * of the other three types.
  */
 #define MAX_TABLE_TYPE 4
```

Comment:
Does this need to be reduced by one now? The max enum now I think will be 3.

Reply:
Good catch.

Reply:
See [7915cc6](https://github.com/wiredtiger/wiredtiger/pull/11278/commits/7915cc61f3d937a92907c9ab7892b8e52b3fc448)